content/articles/getting-started-clickfunnels.markdown

```diff
@@ -0,0 +1,222 @@
+
+    > Hi there,
+
+    > I would like to move my domain(s) from ClickFunnels to my own DNSimple account.
+
+    > i. Confirm that you have made a paid DNSimple account
+
+    > YES
+
+    > ii. Provide the email address associated with your ClickFunnels and DNSimple accounts
+
+    > `your email address`
+
+    > iii. Name of domain(s) you wish to transfer
+
+    > `your domain(s)`
+
+    > iv. Verify the last 4 digits of the card we have on file for your ClickFunnels account
+
+    > `last 4 digits of your credit card`
+
+    > Thanks!
+
+1. ClickFunnels will reply to your email once they have moved the domain into your account. You will also receive an email from us at DNSimple letting you know the domain is waiting for you.
+</div>
+
+
+## Step 3: Accepting the domain and getting it set up in DNSimple
+
+<div class="section-steps" markdown="1">
+1. [Log into DNSimple](https://dnsimple.com/login) where you'll see a <label>Get Started</label> screen. None of these options apply to what you're looking to do. Select <label>Skip this and go to the dashboard</label> found under the three boxes of options. Explore these other options in the [section below](#other-options-when-getting-started-with-a-new-account).
+1. Welcome to the [Dashboard](https://dnsimple.com/dashboard)! This is the homepage of your DNSimple account. You'll see a box to the right that says <label>Alerts</label>. Click <label>View</label> to see your pending domain push and accept it.
+1. You will see <label>Push Requests</label> and your domain listed below. Click <label>Accept</label> to the right of the domain.
+1. Now that you have accepted the push of the domain, you need to change the Registrant information. This information states that you are the owner of the domain. When the domain was with ClickFunnels, they were listed as the owner, which is why we need to update this. See [Updating the registrant information](#updating-the-registrant-information) for a description of each field.
+
+    <note>
+    After the domain is pushed into your DNSimple account, you must also change the ownership of the domain from ClickFunnels to yourself. This [locks the domain for further transfers](/articles/icann-60-day-lock-registrant-change/) for 60 days in accordance with ICANN's rules, meaning that you have to keep your domain here at DNSimple for 60 days before you can transfer the domain elsewhere. Hopefully, you aren't looking to move it but if you are, we will discuss some options later in the article.
+    </note>
+
+    <warning>
+    You will receive a verification email to the email address you used for the domain contact requesting you verify you made a change to the contact information. If you are moving your domain from ClickFunnels, you did make this change and you will want to verify it as soon as you receive the email. Sometimes it lands in spam, so be on the lookout for it.
```

Comment:
I want to include in here that the most common reason someone doesn't receive this email is because of a misspelled address

Reply:
added ✅